src/NServiceBus.Transport.SQS/DelayedMessagesPump.cs

```diff
@@ -413,6 +413,10 @@ async Task DeleteMessage(SqsReceivedDelayedMessage messageToDeleteWithAnotherAtt
                 await sqsClient.DeleteMessageAsync(messageToDeleteWithAnotherAttempt.QueueUrl, messageToDeleteWithAnotherAttempt.ReceiptHandle, cancellationToken).ConfigureAwait(false);
             }
             catch (ReceiptHandleIsInvalidException ex)
+            {
+                Logger.Info($"Message receipt handle '{messageToDeleteWithAnotherAttempt.ReceiptHandle}' is invalid.", ex);
+            }
+            catch (AmazonSQSException ex) when (ex.IsCausedByMessageVisibilityExpiry())
```

Comment:
Shall we link here to the issue about the nuances of these exceptions? This could help our future self remember why two different exceptions are logging the same message.

Reply:
Isn't most of that covered in the PR description?

Reply:
Plus they are not really logging the same message ;)